tenets/codelingo/go/ignore-goroutine-return-values/example.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"time"
+	"fmt"
+)
+
+func main() {
+	go A() // Issue
+        go B() // Non Issue
+	<-time.After(time.Second)
+
+	go func() string {
+	    word := "Hello"
+	    return word // Issue
+        }()
+
+
+
+
+        go func() {
```

Comment:
```suggestion
        go func() { // Issue
```

Reply:
I think you meant `Non Issue` here to replace the one you suggested removing in the above comment